operator/v1/types.go

```diff
@@ -70,11 +70,12 @@ type OperatorSpec struct {
 	// +kubebuilder:default=Normal
 	OperatorLogLevel LogLevel `json:"operatorLogLevel,omitempty"`
 
-	// unsupportedConfigOverrides holds a sparse config that will override any previously set options.  It only needs to be the fields to override
-	// it will end up overlaying in the following order:
-	// 1. hardcoded defaults
-	// 2. observedConfig
-	// 3. unsupportedConfigOverrides
+	// unsupportedConfigOverrides overrides the final configuration that was computed by the operator.
+	// Red Hat does not support the use of this property.
+	// Consider seeking guidance from the Red Hat support team before you use this property,
+	// because misuse of this property could lead to unexpected behavior or conflict with other configuration options.
+	//
+	// Additionally, using this property blocks cluster upgrades, so remove this setting before you upgrade your cluster.
```

Comment:
I had suggested most of this before but it got reverted, I think this paragraph should read as
```suggestion
	// unsupportedConfigOverrides overrides the final configuration that was computed by the operator.
	// Red Hat does not support the use of this field.
	// Misuse of this field could lead to unexpected behavior or conflict with other configuration options.
	// Seek guidance from the Red Hat support before using this field.
	// Use of this property blocks cluster upgrades, it must be removed before upgrading your cluster.
```
Just trying to be concise and direct with the intention of what we are trying to portray here

Reply:
Yes, concise is the best way forward. Less maintenance and chance of causing confusion. My apologies for missing some of those suggested changes. I've implemented them now.